librz/include/rz_util/rz_str.h

```diff
@@ -194,7 +193,7 @@ static inline const char *rz_str_get_null(const char *str) {
 	return str ? str : "(null)";
 }
 RZ_API char *rz_str_ndup(RZ_NULLABLE const char *ptr, int len);
-RZ_API char *rz_str_dup(char *ptr, const char *string);
+RZ_API char *rz_str_dup(const char *str);
```

Comment:
add RZ_NULLABLE for `str` and `RZ_OWN` for the result.

Reply:
same in the .c function

Reply:
Okayy